src/lib/try/TryOnline.svelte

```diff
@@ -74,15 +74,15 @@
         <Angular size={70} color={$darkModeThemeEnabled ? 'currentColor' : "#DD0031"}/>
     </TryTechnology>
 
-    <TryTechnology hrefLink='https://workspaces.openshift.com#https://github.com/devspaces-samples/cakephp-ex/tree/devspaces-3.3-rhel-8&storageType=ephemeral'>
+    <TryTechnology hrefLink='https://workspaces.openshift.com#https://github.com/devspaces-samples/cakephp-ex/tree/devspaces-3.4-rhel-8&storageType=ephemeral'>
```

Comment:
while this URL is valid, the devfile included in 3.4 is now a hello world project that doesn't use composer. 

commit: https://github.com/redhat-developer/devspaces/commit/8bd7eb44865f93f84795dc8a01276c77d7e45d38

new project url: https://github.com/devspaces-samples/php-hello-world/tree/devspaces-3.4-rhel-8
```suggestion
    <TryTechnology hrefLink='https://workspaces.openshift.com#https://github.com/devspaces-samples/php-hello-world/tree/devspaces-3.4-rhel-8&storageType=ephemeral'>
```